### PR TITLE
Switch continuous integration from Travis CI to GitHub Actions

### PR DESCRIPTION
**Pull request by Calinou**
Follow-up to https://github.com/godotengine/godot-docs/pull/3681.

This may conflict with #480. Feel free to merge that one before and I'll rebase.

### CODE REVIEW

**Comment by aaronfranke** [comment]
How does this know to download the `dos2unix` and `recode` packages?

**Comment by Calinou** [comment]
@aaronfranke Fixed. Sorry, I didn't notice it before. I also opened a PR to fix this in godot-docs: https://github.com/godotengine/godot-docs/pull/3705

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+  build:
+    runs-on: ubuntu-20.04
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+
+      - name: Lint demo projects
+        run: |
+          sudo apt-get update -qq
+          sudo apt-get install -qqq dos2unix recode
```
Comment (aaronfranke) [inline .github/workflows/ci.yml:14]: `man` seems to indicate that more than two `q` does nothing:

> More q's will produce more quiet up to a maximum of 2.

**Comment by aaronfranke** [comment]
Merging to be in sync with https://github.com/godotengine/godot-docs/pull/3705. Let's hope it works as expected :)

**Pull request merged**